prisma/schema.prisma

```diff
@@ -62,3 +64,27 @@ model TeamMember {
   @@map("teamMembers")
 }
 
+model Position {
+  id Int @id @default(autoincrement())
+  sport Sport @relation(fields: [sportId], references: [id])
+  sportId Int
+  key String @unique // position name i.e. 'prop'
+  numbers Int[] // jersey numbers for the position. I.e. 'prop' can be [1,3]
```

Comment:
Could make this either a number || number[]

See the seeds for what I mean by this